zerver/lib/soft_deactivation.py

```diff
@@ -11,16 +11,7 @@
 from zerver.models import UserProfile, UserMessage, RealmAuditLog, \
     Subscription, Message, Recipient, UserActivity, Realm
 
-log_format = "%(asctime)s: %(message)s"
-logging.basicConfig(format=log_format)
-
-formatter = logging.Formatter(log_format)
-file_handler = logging.FileHandler(settings.SOFT_DEACTIVATION_LOG_PATH)
-file_handler.setFormatter(formatter)
-
-logger = logging.getLogger('')
-logger.setLevel(logging.INFO)
-logger.addHandler(file_handler)
+logger = create_logger(settings.SOFT_DEACTIVATION_LOG_PATH, 'INFO')
```

Comment:
I just discovered a nasty bug.  The fact that this is using `''` as the logger name meant that basically everything from the server was getting logged to this file, not just the soft-deactivation related code.  I'll fix that and then merge this, after changing `name` to be a mandatory argument.